### PR TITLE
Feature/investment tests

### PR DESCRIPTION
**Pull request by dithh**

### CODE REVIEW

File: test/jest/__tests__/investment/investment.spec.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import {describe, expect, it} from '@jest/globals'
+import investment from 'components/investment/investment'
+
+const defaultInput:InvestmentInputFields = {
+  amount:10000,
+  monthCount:12,
+  rate:0.1,
+}
+
+describe('investment',()=>{
```
Comment (RacyMind) [inline test/jest/__tests__/investment/investment.spec.ts:11]: Generally, it should also check edge cases when one of inputs is equal to 0. I know UI doesn't allow to set 0 but calculations should be correct. The UI might be changed in the future

**Review by RacyMind — CHANGES_REQUESTED** [review]
Please also solve conflicts. I fixed one test and added edge cases for invoice tests

**Comment by dithh** [comment]
@RacyMind  requested changes done

File: test/jest/__tests__/investment/investment.spec.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  it('12 months, 0%, 10000',()=>{
+    const {capital,grossAmount,netAmount,taxAmount} = investment.getResult({...defaultInput,rate:0})
+    expect(capital).toBe(10000)
+    expect(grossAmount).toBe(0)
+    expect(netAmount).toBe(0)
+    expect(taxAmount).toBe(0)
+  })
+
+  it('12 months, 10%, 0',()=>{
+    const {capital,grossAmount,netAmount,taxAmount} = investment.getResult({...defaultInput,rate:0})
```
Comment (RacyMind) [inline test/jest/__tests__/investment/investment.spec.ts:61]: `amount` should be here :)

**Review by RacyMind — CHANGES_REQUESTED** [review]

File: test/jest/__tests__/investment/investment.spec.ts (hunk 2 above)
Comment (dithh) [reply test/jest/__tests__/investment/investment.spec.ts]: done

**Review by RacyMind — APPROVED** [review]
